sdk/src/camera_96tof1_specifics.cpp

```diff
@@ -50,6 +50,13 @@ uint16_t Camera96Tof1Specifics::noiseReductionThreshold() const {
 }
 
 Status Camera96Tof1Specifics::setTresholdAndEnable(uint16_t treshold, bool en) {
+    aditof::CameraDetails cameraDetails;
+    m_camera->getDetails(cameraDetails);
+    if (cameraDetails.mode.compare("far") == 0) {
+        LOG(ERROR) << "Far mode does not support noise reduction";
```

Comment:
This is treated too severely. A warning should be enough.

Reply:
solved. thank you